inference-engine/src/vpu/graph_transformer/include/vpu/frontend/frontend.hpp

```diff
@@ -155,6 +155,8 @@ class FrontEnd final {
     void parseGelu(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const;
     void parseSoftPlus(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const;
     void parseSwish(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const;
+    void parseActivation(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const;
+    void parseLogicalNot(const Model& model, const ie::CNNLayerPtr& layer, const DataVector& inputs, const DataVector& outputs) const;
```

Comment:
No code in this scope uses `parseLogicalNot`. Should we declare it here?

Reply:
I'm not sure I understand your point. parseActivation uses parseLogicalNot

Reply:
I proposed do not declare `parseLogicalNot` in `frontend.hpp` because of `parseLogicalNot` is not called from `frontend.cpp` but in `activation.cpp`

Reply:
Discussed offline. No changes are needed.